org.uqbar.project.wollok.typeSystem/src/org/uqbar/project/wollok/typesystem/WollokTypeSystemActivator.xtend

```diff
@@ -55,6 +57,11 @@ class WollokTypeSystemActivator extends Plugin {
 		getTypeSystem(selectedTypeSystem)
 	}
 
+	def getTypeSystem(IProject project) {
```

Comment:
This seems to have some duplication with https://github.com/uqbar-project/wollok/pull/1350/files?utf8=%E2%9C%93&diff=split&w=1#diff-79a07cfa0bc00ffd6fa5766436ffcc65R29